src/test/groovy/com/stormpath/tck/logout/LogoutIT.groovy

```diff
@@ -0,0 +1,320 @@
+ * limitations under the License.
+ */
+package com.stormpath.tck.logout
+
+import com.jayway.restassured.http.ContentType
+import com.jayway.restassured.response.Cookie
+import com.jayway.restassured.response.Cookies
+import com.stormpath.tck.AbstractIT
+import com.stormpath.tck.util.EnvUtils
+import com.stormpath.tck.util.JwtUtils
+import com.stormpath.tck.util.RestUtils
+import org.testng.annotations.BeforeClass
+import org.testng.annotations.Test
+
+import static com.jayway.restassured.RestAssured.*
+import static org.testng.Assert.*
+import static org.hamcrest.Matchers.*
+
+@Test
+class LogoutIT extends AbstractIT {
+
+    private final String randomUUID = UUID.randomUUID().toString()
+    private final String accountEmail = "fooemail-" + randomUUID + "@stormpath.com"
+    private final String accountGivenName = "GivenName-" + randomUUID
+    private final String accountSurname = "Surname-" + randomUUID
+    private final String accountPassword = "P@sword123!"
+
+    private final String logoutPath = "/logout"
+    private final String registerPath = "/register"
+    private final String loginPath = "/login"
+
+    @BeforeClass
+    public void createTestUser() throws Exception {
+
+        Map<String, Object>  jsonAsMap = new HashMap<>();
+        jsonAsMap.put("email", accountEmail)
+        jsonAsMap.put("password", accountPassword)
+        jsonAsMap.put("givenName", accountGivenName)
+        jsonAsMap.put("surname", accountSurname)
+
+        String createdHref =
+            given()
+                .accept(ContentType.JSON)
+                .contentType(ContentType.JSON)
+                .body(jsonAsMap)
+            .when()
+                .post(registerPath)
+            .then()
+                .statusCode(200)
+            .extract()
+                .path("account.href")
+
+        deleteOnClassTeardown(createdHref)
+    }
+
+    private Map<String, String> createSession() throws Exception {
+
+        Map<String, Object>  credentials = new HashMap<>();
+        credentials.put("login", accountEmail)
+        credentials.put("password", accountPassword)
+
+        Map<String, String> cookies =
+            given()
+                .accept(ContentType.JSON)
+                .contentType(ContentType.JSON)
+                .body(credentials)
+            .when()
+                .post(loginPath)
+            .then()
+                .statusCode(200)
+            .extract()
+                .cookies()
+
+        return cookies
+    }
+
+    private void assertCookiesAreDeleted(Cookies cookies) throws Exception {
+        Date currentTime = new Date(System.currentTimeMillis())
+        Date today = currentTime.copyWith(
+            hourOfDay: 0,
+            minute: 0,
+            second: 0
+        )
+
+        Cookie accessTokenCookie = cookies.get("access_token")
+        assertTrue(accessTokenCookie.value.isEmpty())
+        assertTrue(accessTokenCookie.expiryDate < today)
```

Comment:
The test isn't checking for epoch, it's checking that it's set to earlier than today.